notebooks/COS/Setup/Setup.ipynb

```diff
@@ -297,7 +297,7 @@
    "metadata": {},
```

Comment:
<p>Sierra, I know you had these fixed on the closed PR. I would like to remind you of this. Since the closed PR wasn't merged, you would need to do the same updates again. This <strong>ViewData.ipynb </strong>still references the older repo. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>